app/src/main/java/com/starry/greenstash/ui/navigation/NavGraph.kt

```diff
@@ -51,26 +50,25 @@ import com.starry.greenstash.ui.screens.welcome.composables.WelcomeScreen
 @Composable
 fun NavGraph(
     navController: NavHostController,
-    startDestination: String
+    startDestination: Any
```

Comment:
Instead of using `Any`, I think it would be much better to create a supertype, like a class called `Screen`, which both drawer screens and normal screens can inherit from. The supertype doesn't need to accept any parameters, and we can override the constructor in specific screen types where parameters are needed, such as in drawer screens or others. Doing so will give us the benefit of type safety and static type checking at compile time. On the other hand, using `Any` allows passing anything, like strings, integers, or something else, instead of an actual screen. This won't be detected at compile time and could lead to unexpected bugs or crashes at runtime later on.

Reply:
@starry-shivam Thanks man for your constructive feedback

Reply:
@starry-shivam I have changed the Any type to type safety and static type checking at compile time. Also Updated the code please let me know your valuable feedback if i am missing something.
Thanks